cli/src/main.rs

```diff
@@ -84,6 +86,22 @@ fn build_command() -> Command<'static> {
                 .takes_value(true)
                 .required(false),
         )
+        .arg(
+            Arg::new(ARG_SCALA_PACKAGE)
+                .short('S')
```

Comment:
Honestly I think we could go without the short args here, especially since it's really unclear what this is an abbreviation for at a glance. Also, the lowercase version is for Swift prefixes, which is totally unrelated and having the capitalized version be a config for a different language _and_ option is confusing.

Reply:
I agree, I just couldn't think of anything clearer and wanted to follow the previous convention of including short args.